emacs.d/b-python.el

```diff
@@ -2,11 +2,20 @@
 (provide 'b-python)
 (require 'python)
 
-(defvar b-python-copyright-owner "RadiaSoft LLC."
-  "*Who owns the copyrights for templates")
+(defcustom b-python-fmt-file-regexp (expand-file-name "~/src/radiasoft/")
+  "What Python files should be automatically set"
```

Comment:
I find this confusing. I'm not sure what set means in this context?

"What Python files to format"?

Reply:
[84adf91](https://github.com/biviosoftware/home-env/pull/116/commits/84adf91dfa587d14d47151f97910276060d5c5ff)